### ISSUE TITLE
Bug: Do not offer to call an offline person in DC

### ISSUE THREAD

**Issue by Cyborgscode** [SchildiChat/schildichat-desktop#49]
It's possible to start a call in a DC to an offline person.

Which is in no way clear, when you hit the button. 

I suggest to at least open a "are you sure" request with a hint, that this contact is offline.

**Comment by ShadowJonathan** [comment]
(DC = "direct chat"?)

Also, presence is disabled on many public matrix servers atm, so there's no knowing who is an "offline" person most of the time, and even if a person was "offline", they could just be broadcasting their presence as such.

**Comment by Cyborgscode** [comment]
"Offline": People use it to not get contacted, so: Mission accomplished. 

Why did i open this as bug: Our HS has presence enabled, person was rellay offline, call happend against all odds of ever fullfilling.

**Comment by ShadowJonathan** [comment]
I dont think this is needed, you can just see for yourself if a person is "offline", if the problem is that you cant see that clearly, thats a UI/UX issue on presence, which indeed isn't supported that widely.

**Comment by su-ex** [comment]
Android app seems to ask if you really want to call somebody independent of presence. Did I get this right? Would a similar confirmation dialogue independent of presence before really calling somebody fulfill your needs?
